sdk/communication/Azure.Communication.PhoneNumbers/tests/Infrastructure/PhoneNumbersClientTestEnvironment.cs

```diff
@@ -1,11 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Azure.Communication.Tests;
 
 namespace Azure.Communication.PhoneNumbers.Tests
 {
     public class PhoneNumbersClientTestEnvironment : CommunicationTestEnvironment
     {
+        internal const string LiveTestConnectionStringEnvironmentVariableName = "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING";
```

Comment:
The string is common across all sdks, right? Can we move this variable to the parent class CommunicationTestEnvironment.

Reply:
Yup, good idea Sam